mix.exs

```diff
@@ -34,11 +34,9 @@ defmodule GRPC.Mixfile do
 
   defp deps do
     [
-      {:protobuf, "~> 0.5"},
+      {:protobuf, ">= 0.5.0 and < 0.10.0"},
```

Comment:
How did you arrive at this selector?

Reply:
Version 0.10.0 introduces breaking changes that require a bigger update in the `grpc` library - the same breaking changes that prevent us from using the upstream `grpc` library I'm guessing.